### ISSUE TITLE
"Preview" in Mercury is busted

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-cms#31]
_From @adamjohnson on February 13, 2014 0:7_

### Steps to Reproduce

1. Edit any page in CleanSlate.
1. Hit the "Preview" button in Mercury.

Result: Nothing changes.

### Expected Behavior:

Editable region fields should disappear and you should be able to see how your page would look to normal site visitors (minus the Mercury edit bar).

Here's the error in the console:

```
Uncaught TypeError: Object [object global] has no method '$' editor:155
(anonymous function) editor:155
jQuery.extend.trigger mercury-a57d20e645b7e6076de77be6beaf63bb.js:32
Mercury.Toolbar.Button.t.handleClick mercury-a57d20e645b7e6076de77be6beaf63bb.js:33
(anonymous function) mercury-a57d20e645b7e6076de77be6beaf63bb.js:33
```

Tested in Chrome and Firefox.

_Copied from original issue: wvuweb/cleanslate#30_

**Comment by adamjohnson** [comment]
_From @nreckart on February 13, 2014 17:13_

Can't reproduce. Works fine for me.

**Comment by adamjohnson** [comment]
_From @nreckart on February 13, 2014 18:30_

This only seems to be an issue when editing a site that does not include jQuery in its theme. [Some JS](https://github.com/wvuweb/cleanslate/blob/dev/app/views/layouts/mercury.html.erb#L70-L71) associated with the Gist and Comment snippets seem to be the cause of the error.

**Comment by adamjohnson** [comment]
_From @zeroedin on February 13, 2014 18:33_

The [Some JS](https://github.com/wvuweb/cleanslate/blob/dev/app/views/layouts/mercury.html.erb#L70-L71) was more then likely a missunderstanding of mine on the context of where the JS was running.  In this case I would assume that it expects the iframe to have JQuery.  Might just have to be rewritten in JS that does not assume JQuery.

**Comment by adamjohnson** [comment]
Fixed in d4fc2c73e8822eedbec520a4371d387de50b01bc.
